cmd/snap-confine/udev-support.c

```diff
@@ -116,6 +116,34 @@ static void sc_udev_allow_nvidia(sc_device_cgroup *cgroup)
 	}
 }
 
+/** Allow access to hybris devices.
+ *
+ * Required by Halium-based GNU/Linux adaptations to make use of certain device nodes.
+ *
+ * Note: Binder devices on newer Android kernels reside inside of their own binderfs mountpount.
+ **/
+static void sc_udev_allow_hybris(sc_device_cgroup *cgroup)
+{
+	struct stat sbuf;
+
+	if (stat("/dev/binderfs/binder", &sbuf) == 0) {
+		sc_device_cgroup_allow(cgroup, S_IFCHR, major(sbuf.st_rdev),
+				       minor(sbuf.st_rdev));
+	}
+	if (stat("/dev/binderfs/hwbinder", &sbuf) == 0) {
+		sc_device_cgroup_allow(cgroup, S_IFCHR, major(sbuf.st_rdev),
+				       minor(sbuf.st_rdev));
+	}
+	if (stat("/dev/binder", &sbuf) == 0) {
```

Comment:
Since this runs everywhere, what happens if someone installs, say, anbox and gets /dev/binder. Now all snaps will allow access to the binder. I don't have an immediate idea on how to make this better without some host-level hint.

Reply:
The host level hint can again be a check for existance of /system/build.prop, which under Halium distributions always has to exist.

Reply:
Handled in d5a85d6